discovery-infra/tests/test_cancel_and_reset.py

```diff
@@ -44,10 +44,11 @@ def test_cancel_reset_before_node_boot(self, api_client, node_controller):
         self.wait_until_hosts_are_discovered(cluster_id=cluster_id, api_client=api_client)
         self.wait_until_cluster_is_ready_for_install(cluster_id=cluster_id, api_client=api_client)
         # Install Cluster
-        self.start_cluster_install(cluster_id=cluster_id, api_client=api_client)
-        # wait until all nodes are in Installed status, will fail in case one host in error
-        self.wait_for_nodes_to_install(cluster_id=cluster_id, api_client=api_client)
-        self.wait_for_cluster_to_install(cluster_id=cluster_id, api_client=api_client)
+        # TODO need to think how to test it and make it quick
+        # self.start_cluster_install(cluster_id=cluster_id, api_client=api_client)
```

Comment:
Move this part from both tests to an helper function

Reply:
thought u will do it. it must be part of cluster controller. Not now